### PR TITLE
Add Support for Syntax Highlighting

### LINKED ISSUE #14: Syntax Highlighting
Adding Support for Syntax Highlighting would be much useful. I implemented one on my personal fork with [chroma](https://github.com/alecthomas/chroma) and it works perfectly. I also had an idea on customizing the colorschemes from the `zettel.toml` file, and found a rather crooked workaround with golang maps.
```go
colorScheme := map[string]*chroma.Style{
		"Monokai":      styles.Monokai,
		"MonokaiLight": styles.MonokaiLight,
	}
```
This has to be finished with all the colorschemes of chroma. The current build works fine. Need some feedback for he above.

I'll send a PR soon

### PR DESCRIPTION
**Pull request by athul**
Uses Chroma for the Color ✨ 

The new config generates a new field called `pygmentsstyle`(just like Hugo 😉  ).

The only limitation I found is that you have to define the colorscheme in small letters. You can find all the styles [here](https://xyproto.github.io/splash/docs/).

Had to tweak the CSS a bit to give a better BG color of the code block

Fixes #14

### CODE REVIEW

**Comment by mr-karan** [comment]
`wing.css` is the framework we use. All the overrides should go to our custom stylesheet.

**Comment by athul** [comment]
Oh Okay, Will Change that

**Comment by athul** [comment]
e4aec4172b46cdd30e76e4011e194cc6853a0c21 changes the usage to custom sheet.

Here is an output for reference. Syntax Used: GitHub

Screen Shot 2020-09-16 at 3 07 48 PM

File: internal/pipeline/utils.go [hunk 1]
```diff
@@ -17,3 +24,48 @@ func findLinks(body string) []string {
 	}
 	return matches
 }
+
+//SyntaxHighlighter Highlights any code inside the MD Files
+func SyntaxHighlighter(html []byte, syntaxStyle string) string {
```
Comment (mr-karan) [inline internal/pipeline/utils.go]: Can you add some comments in this function, how does this really work?

**Review by mr-karan — CHANGES_REQUESTED** [review]

**Comment by mr-karan** [comment]
Thanks @athul  :+1:

**Pull request merged**